frontend/app/application.py

```diff
@@ -280,7 +280,10 @@ def human_name_filter(person):
     name = person.get('givenNames') or ''
     if 'nameParticle' in person and person['nameParticle']:
         name += ' ' + person.get('nameParticle', '')
-    return name + ' ' + person.get('familyNames', '')
+    name += ' ' + person.get('familyNames', '')
+    if 'nameSuffix' in person and person['nameSuffix']:
+        name += ' ' + person.get('nameSuffix', '')
+    return name
```

Comment:
not exactly equivalent but what do you think about something like:
```python
nameparts = [person.get("givenNames", ""),
             person.get("nameParticle", ""),
             person.get("familyNames", ""),
             person.get("nameSuffix", "")]

return " ".join(nameparts)
```

Reply:
Nope tests says no,

```shell
FAILED tests/test_template_filters.py::test_human_name_filter[person0-x y] - AssertionError: assert 'x  y ' == 'x y'
FAILED tests/test_template_filters.py::test_human_name_filter[person1-x de y] - AssertionError: assert 'x de y ' == 'x de y'
FAILED tests/test_template_filters.py::test_human_name_filter[person2-x y Jr.] - AssertionError: assert 'x  y Jr.' == 'x y Jr.'
```